src/docs/release/breaking-changes/material-chip-button-semantics.md

```diff
@@ -0,0 +1,156 @@
+subpar experience, unless you implemented a workaround by manually adding the
+missing semantics to the Chip widgets in your app.
+
+## Description of change
+
+The outermost [`Semantics`][] widget which wraps all Chip classes to describe
+their semantic properties has been modified.
+
+For [`ActionChip`][], [`ChoiceChip`][], [`FilterChip`][], and [`InputChip`][]:
+
+- The [`button`][`SemanticsProperties.button`] property is set to `true`.
+- The [`enabled`][`SemanticsProperties.enabled`] property reflects whether the
+  Chip is _currently_ tappable (by having a callback set).
+
+These property changes bring interactive Chips' semantic behavior in-line with
+that of other [Material Buttons][].
+
+For the non-interactive information [`Chip`][]:
+
+- The [`button`][`SemanticsProperties.button`] property is set to `false`.
+- The [`enabled`][`SemanticsProperties.enabled`] property is set to `null`.
+
+## Migration guide
+
+**You might not need to perform any migration.** This change only affects you if
+you worked around the issue of Material Chips missing `button` semantics by
+wrapping the widget given to the `label` field of a `Chip` with a `Semantics`
+widget marked as `button: true`. **In this case, the inner and outer `button`
+semantics will conflict, resulting in the tappable area of the button shrinking
+down to the size of the label after this change is introduced.** This issue can
+be fixed by deleting that `Semantics` widget and replacing it with its child, or
+simply removing the `button: true` property if other semantic properties still
+need to be applied to the `label` widget of the Chip.
+
+The following snippets use [`InputChip`][] as an example, but the same process
+applies to [`ActionChip`][], [`ChoiceChip`][], and [`FilterChip`][] as well.
+
+**Case 1: remove the `Semantics` widget.**
+
+Code before migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Semantics(
+    button: true,
+    child: Text('My Input Chip'),
+  ),
+);
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Text('My Input Chip'),
+);
+```
+
+**Case 2: remove `button:true` from the `Semantics` widget.**
+
+Code before migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Semantics(
+    button: true,
+    hint: 'Example Hint',
+    child: Text('My Input Chip'),
+  ),
+);
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+Widget myInputChip = InputChip(
+  onPressed: () {},
+  label: Semantics(
+    hint: 'Example Hint',
+    child: Text('My Input Chip'),
+  ),
+);
+```
+
+## Timeline
+
+Landed in version: TBD<br> In stable release: not yet
+
+## References
+
+{% include master-api.md %}
+
+API documentation:
+* [`ActionChip`][]
+* [`Chip`][]
+* [`ChoiceChip`][]
+* [`FilterChip`][]
+* [`InputChip`][]
+* [Material Buttons][]
```

Comment:
Should this link point to {{site.material}}, like the next one does? (I was surprised that two links that looked similar didn't go to similar places.)

(If this should go to site.material but the earlier usage of [Material Buttons] shouldn't, then use a new link name for the earlier one.)

Reply:
Good catch! Updated it to `{{site.material}}`.